src/Homie/Helpers.cpp

```diff
@@ -18,13 +18,13 @@ const char* Helpers::getDeviceId() {
 MdnsQueryResult Helpers::mdnsQuery(const char* service) {
   MdnsQueryResult result;
   result.success = false;
-  int n = MDNS.queryService(service, "tcp");
+  int n = 0; // MDNS.queryService(service, "tcp");
```

Comment:
@euphi I fixed mDNS support in Homie and I've it working flawlessly in my lab for more than one week. Please, do not comment it. If you keep getting resets, raise a new bug report, please.